### DISCUSSION TITLE
Don't receive email on sign up/login

### DISCUSSION THREAD

**Discussion by evamalininaa** [pockethost/pockethost#109] [Q&A]
Creating new app, cant auth, cant receive email. Is there bug ?

**Comment by benallfree** [comment]
Hi @evamalininaa, are you referring to your PocketHost site login, or your PocketBase instance admin login? 

I just tested pockethost.io and everything is working correctly - I can sign up, receive a verification email, and log in. 

Please let me know if you have further trouble or details on what is not working.

**Comment by evamalininaa** [comment]
Hi I don’t receive verification mail. I tested 2 email (gmail, outlook)

**Reply by benallfree** [reply]
Please email ben@pockethost.io and I’ll help you more. I need to look up your account.

**Reply by evamalininaa** [reply]
just wrote you, thanks

**Comment by irakli2206** [comment]
For some reason I had this issue too, using gmail. I just had to go into 'All Mail' section above Spam on the left and the mail was there.
